### DISCUSSION TITLE
CNC Genmitsu 4040-pro won't connect to UGS

### DISCUSSION THREAD

**Discussion by my4040** [winder/Universal-G-Code-Sender#2523] [Help]
Hi, 
I'm trying to connect my CNC machine (Genmitsu 4040-pro) but I'm constantly getting an error message (see attached file). I'm running Win11 with the latest USG software. I also went through different troubleshooting steps (cables, restart, only one CNC machine running with only one CNC software, etc.) but it's still not working for me.

It's my first time using USG, your help is greatly appreciated.

![error](https://github.com/winder/Universal-G-Code-Sender/assets/170452679/1c1fdcf9-2e80-4317-ab2b-7c20d38875e6)

**Comment by breiler** [comment]
I am guessing that you figured it out as you've managed to connect to the machine here: #2524
